media/js/mozorg/home/home.js

```diff
@@ -66,4 +70,68 @@
 
     // Video card interactions.
     initVideoEvents();
-})();
+
+    /*
+     * Sticky CTA
+     */
+
+
+    var $stickyCTA = $(document.getElementById('download-firefox-sticky-cta'));
+    var hasCookies = typeof Mozilla.Cookies !== 'undefined' && Mozilla.Cookies.enabled();
+    $stickyCTA.attr('aria-hidden', 'true');
+
+    // init dismiss button
+    function initStickyCTA() {
+        // add and remove aria-hidden
+        var primaryTop = new Waypoint({
+            element: document.getElementById('download-firefox-primary-cta'),
+            handler: function(direction) {
+                if(direction === 'down') {
+                    // becomes percivable as the user scrolls down
+                    $stickyCTA.removeAttr('aria-hidden');
+                } else {
+                    // hidden again as they scroll up
+                    $stickyCTA.attr('aria-hidden', 'true');
+                }
+            }
+        });
+
+        // add button
+        var $dismissButton = $('<button>').addClass('sticky-dismiss').text('Dismiss download prompt.');
+        var $stickyWrapper = $stickyCTA.find('.primary-wrapper');
+        $dismissButton.appendTo($stickyWrapper);
+        // listen for click
+        $dismissButton.one('click', function(){
+            // dismiss
+            dismissStickyCTA(primaryTop);
+        });
+    }
+
+    // handle dismiss
+    function dismissStickyCTA(wayPoint) {
```

Comment:
The variable name `wayPoint` scares me a little because it's so close to conflicting with `Waypoint`. Not critical to change for this test, but 😬